src/Features/RemotePaymentMethods/Init.php

```diff
@@ -53,15 +53,46 @@ public static function get_specs() {
 
 		// Fetch specs if they don't yet exist.
 		if ( false === $specs || ! is_array( $specs ) || 0 === count( $specs ) ) {
-			// We are running too early, need to poll data sources first.
+			if ( 'no' === get_option( 'woocommerce_show_marketplace_suggestions', 'yes' ) ) {
+				return self::get_default_specs();
+			}
+
 			$specs = DataSourcePoller::read_specs_from_data_sources();
+
+			// Fall back to default specs if polling failed.
+			if ( ! $specs ) {
+				return self::get_default_specs();
+			}
+
 			$specs = self::localize( $specs );
 			set_transient( self::SPECS_TRANSIENT_NAME, $specs, 7 * DAY_IN_SECONDS );
 		}
 
 		return $specs;
 	}
 
+	/**
+	 * Get default specs.
+	 *
+	 * @return array Default specs.
+	 */
+	public static function get_default_specs() {
+		return array(
```

Comment:
How are we going to manage these defaults going forward? Will we try to sync them with the remote configuration, but not reflecting the most up-to-date changes?

Reply:
I'm not entirely sure.  We can continue the discussion for defaults in https://github.com/woocommerce/woocommerce-admin/issues/6855 but I'm in favor of keeping them minimal and as fallbacks only.  I think continuing to update with suggestions attempts to circumvent opting out of marketplace suggestions and increases code duplication.

Reply:
The idea has been to update to reflect changes in the remote config, but a broader discussion would be useful